packages/core/__test__/utils/elementReady.test.js

```diff
@@ -55,8 +55,8 @@ describe('TestReady', function () {
 
     const { errorMessage, errorCount } = getErrors();
 
-    expect(errorMessage).to.equal('test');
-    expect(errorCount).to.equal(1);
+    expect(errorCount).to.equal(1, 'Error not thrown');
+    await expect(errorMessage).to.equal('test');
```

Comment:
This looks incomplete. Did you forget to push your change?

Reply:
Fixed